### PR TITLE
Describe more chain call behaviours

### PR DESCRIPTION
**Pull request by vaivaswatha**

### CODE REVIEW

File: docs/source/scilla-in-depth.rst [hunk 1]
```diff
@@ -2258,6 +2258,12 @@ An example ``init.json``:
 Chain Invocation Behaviour
 **************************
 
+In a sequence of contract calls (i.e, a contract transition execution
+resulting in the execution of one or more similar transition executions
+(of the same or other contracts), any kind of failure at one point will
+result in the entire set of executions to be discarded (except for the
+gas already consumed).
+
```
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst]: LGTM
Could you also add a sentence on the estimate (a range, perhaps, as it varies somewhat as the blockchain evolves) of the allowed length of chain calls?

Comment (AmritKumar) [reply docs/source/scilla-in-depth.rst]: LGTM

Comment (AmritKumar) [reply docs/source/scilla-in-depth.rst]: What I think is missing is some clarity on on the semantics of `accept`. What happens when a contract is sent money but it does not have accept, does the transaction get rejected or the transaction is processed except that the incoming money is rejected. @vaivaswatha This is not necessarily a chain-call topic but something that we should clarify for `accept` statement.

Comment (anton-trunov) [reply docs/source/scilla-in-depth.rst]: Looks like there is an attempt to explain the semantics of `accept` here: https://scilla.readthedocs.io/en/latest/scilla-in-depth.html#statements. 

>If a message contains QA, but the invoked transition does not accept the money, the money is transferred back to the sender of the message.

It does not mention transition being rejected, so I gather it does not, but you are right -- we should explicitly say so.

Comment (vaivaswatha) [reply docs/source/scilla-in-depth.rst]: > LGTM
> Could you also add a sentence on the estimate (a range, perhaps, as it varies somewhat as the blockchain evolves) of the allowed length of chain calls?

Done.

Comment (vaivaswatha) [reply docs/source/scilla-in-depth.rst]: > Looks like there is an attempt to explain the semantics of `accept` here: https://scilla.readthedocs.io/en/latest/scilla-in-depth.html#statements.
> 
> > If a message contains QA, but the invoked transition does not accept the money, the money is transferred back to the sender of the message.
> 
> It does not mention transition being rejected, so I gather it does not, but you are right -- we should explicitly say so.

Done

**Pull request merged**